README.md

```diff
@@ -51,15 +51,61 @@ Participation in the Kubernetes community is governed by the [Kubernetes Code of
 
 ```bash
 go get github.com/kubernetes-incubator/cri-tools/cmd/critest
-$GOPATH/bin/critest
+go get github.com/kubernetes-incubator/cri-tools/cmd/critctl
 ```
+### critest
 
-### Flags
+ validation test suites for kubelet CRI
 
-`--runtime-service-address`, `-r`:The default server is dockershim. If we want to test other CRI server such as frakti, we can add flag `--runtime-service-address=/var/run/frakti.sock`. And we can run this test against frakti.
+#### Flags
 
-`--build-dependencies`, `-b`:If we don't need to build dependencies, we can add this flag `--build-dependencies=false` or `-b=false`.
+- `--runtime-service-address`, `-r`:The default server is dockershim. If we want to test other CRI server such as frakti, we can add flag `--runtime-service-address=/var/run/frakti.sock`. And we can run this test against frakti.
```

Comment:
missing image service address. Could you also change them to `--runtime-endpoint` and `--image-endpoint`? (consistent with crictl)